packages/aws-amplify-react-native/src/InAppMessaging/components/ModalMessage/__tests__/Modal.spec.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2017-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+
+import { IN_APP_MESSAGING } from '../../../../AmplifyTestIDs';
+import useMessageImage from '../../hooks/useMessageImage';
+import { INITIAL_IMAGE_DIMENSIONS } from '../../hooks/useMessageImage/constants';
+
+import ModalMessage from '../ModalMessage';
+
+jest.mock('../../hooks/useMessageImage', () => ({
+	__esModule: true,
+	default: jest.fn(),
+}));
```

Comment:
Nit: I *think* you should be able to just let it jest automock this
```suggestion
jest.mock('../../hooks/useMessageImage');
```